src/scaffolding/wizard/netCore/NetCoreGatherInformationStep.ts

```diff
@@ -76,4 +81,45 @@ export class NetCoreGatherInformationStep extends GatherInformationStep<NetCoreS
     protected setTelemetry(wizardContext: NetCoreScaffoldingWizardContext): void {
         wizardContext.telemetry.properties.netCoreVersion = this.targetFramework;
     }
+
+    private async ensureNetCoreBuildTasks(wizardContext: NetCoreScaffoldingWizardContext): Promise<void> {
+        const cSharpExtension: vscode.Extension<CSharpExtensionExports> | undefined = vscode.extensions.getExtension(cSharpExtensionId);
+        if (hasTask('build', wizardContext.workspaceFolder) && cSharpExtension) {

```

Comment:
TODO: Enforce a minimum version, also wait for the upstream fix we need to be released